src/Elastic.Apm/ServerInfo/IServerInfo.cs

```diff
@@ -0,0 +1,36 @@
+// Licensed to Elasticsearch B.V under
+// one or more agreements.
+// Elasticsearch B.V licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information
+
+using System;
+using System.Threading.Tasks;
+
+namespace Elastic.Apm.ServerInfo
+{
+	/// <summary>
+	/// Encapsulates information about the APM Server that receives data from the agent.
+	/// </summary>
+	internal interface IServerInfo
```

Comment:
Might be worth naming this `IApmServerInfo` to indicate that it's information specific to APM server, as opposed to any other server e.g. server on which agent is running

Reply:
Agreed - implemented.